qiskit/algorithms/minimum_eigensolvers/vqe.py

```diff
@@ -308,11 +310,13 @@ def _check_operator_ansatz(self, operator: BaseOperator | PauliSumOp):
 
     def _build_vqe_result(
         self,
+        ansatz: QuantumCircuit,
         optimizer_result: OptimizerResult,
         aux_operators_evaluated: ListOrDict[tuple[complex, tuple[complex, int]]],
         optimizer_time: float,
     ) -> VQEResult:
         result = VQEResult()
+        result.ansatz = ansatz
```

Comment:
This should be copied, otherwise if a user changes the ansatz it will also change in the result 🙂